demo/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.example</groupId>
+  <artifactId>demo</artifactId>
+  <version>1.0</version>
+
+  <properties>
+    <spring-boot.version>2.7.5</spring-boot.version>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <!-- Instead of using spring-boot-starter-parent as POM parent we add
+      pieces needed from it; also see:
+      https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#using.import
+      -->
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${spring-boot.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.fhir.gateway</groupId>
+      <artifactId>server</artifactId>
+      <version>0.1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.fhir.gateway</groupId>
+      <artifactId>plugins</artifactId>
```

Comment:
Do we need this dep because you want to give the user the option for choosing the default access-checkers too (e.g., through command-line args)?

Reply:
yes I wanted to add this so that I can have the right configuration for loading this. 
eg - just adding this is not enough. We need to add the package to scanBasePackages of SpringBootApplication